README.md

```diff
@@ -45,152 +21,53 @@ though we include a summary of them here:
 * Binary fixed-point math
 * Restrictions on reentrancy
 * Static looping
+* Module imports
+* Usage of [YUL](https://docs.soliditylang.org/en/latest/yul.html) IR to target both EVM and eWASM
+* WASM compiler binaries for enhanced portability and in-browser compilation of
+  Fe contracts
+* Implementation in a powerful, systems-oriented language (Rust) with strong safety guarantees to reduce risk of compiler bugs
```

Comment:
@g-r-a-n-t  I unified the project goals in this section. Let me know if you think I left out something important. I left out thinks that we aren't yet fully decided on such as inheritance or modifier